### PR TITLE
Add migration docs yiisoft/db-migration

### PR DESCRIPTION
**Pull request by hacan359**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: guide/en/databases/db-migrations.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    public function down(MigrationBuilder $b): void
+    {
+        $b->dropTable('my_first_table');
+    }
+}
+```
+
+For more information [see](https://github.com/yiisoft/db-migration/tree/master/docs/en)
+
+### Migration from YII2
```
Comment (samdark) [inline guide/en/databases/db-migrations.md:90]:
```suggestion
### Upgrading from Yii2
```

File: guide/en/databases/db-migrations.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# Migrations
+
+To use migrations, install [yiisoft/db-migration](https://github.com/yiisoft/db-migration/) package:
+
+```shell
+composer require yiisoft/db-migration
+```
+
+### Example config with Yii Console
```
Comment (samdark) [inline guide/en/databases/db-migrations.md:9]: The context of the guide assumes that we're using Yii console so there's no need to mention it specifically.

File: guide/en/databases/db-migrations.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    ConnectionInterface::class => [
+        'class' => SqliteConnection::class,
+        '__construct()' => [
+            'dsn' => 'sqlite:' . __DIR__ . '/Data/yiitest.sq3'
+        ]
+    ]
+];
+```
+
+Add to `config/params.php`:
```
Comment (samdark) [inline guide/en/databases/db-migrations.md:31]:
```suggestion
Add the following to `config/params.php`:
```

File: guide/en/databases/db-migrations.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+```php
+...
+'yiisoft/db-migration' => [
+    'newMigrationNamespace' => 'App\\Migration',
+    'sourceNamespaces' => ['App\\Migration'],
+],
+...
+```
+
```
Comment (samdark) [inline guide/en/databases/db-migrations.md:41]:
```suggestion

Not test if it works:
```

File: guide/en/databases/db-migrations.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+```shell
+./yii list migrate
+```
+
+### Example create migrate
```
Comment (samdark) [inline guide/en/databases/db-migrations.md:46]:
```suggestion
### Creating a migration
```

File: guide/en/databases/db-migrations.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+To work with migrations, you can use the provided [view](https://github.com/yiisoft/db-migration/tree/master/resources/views).
+
+```shell
+./yii migrate:create my_first_table --command=table --fields=name,example --table-comment=my_first_table
+```
+
+```php
```
Comment (samdark) [inline guide/en/databases/db-migrations.md:54]:
```suggestion

That would generate the following:

```php
```

File: guide/en/databases/db-migrations.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+        $b->dropTable('my_first_table');
+    }
+}
+```
+
+For more information [see](https://github.com/yiisoft/db-migration/tree/master/docs/en)
+
+### Migration from YII2
+
+Migrations in yii2 and the [yiisoft/db-migration](https://github.com/yiisoft/db-migration/) package are not compatible,
```
Comment (samdark) [inline guide/en/databases/db-migrations.md:92]:
```suggestion
Migrations in Yii2 and the [yiisoft/db-migration](https://github.com/yiisoft/db-migration/) package are not compatible,
```

File: guide/en/databases/db-migrations.md (hunk 2 above)
Comment (hacan359) [reply guide/en/databases/db-migrations.md]: done

**Comment by samdark** [comment]
Thanks!

**Pull request merged**